packages/graphql/lib/mock-server-middleware.js

```diff
@@ -0,0 +1,100 @@
+const { ApolloServer } = require('apollo-server-express');
+const cookieParser = require('cookie-parser');
+const express = require('express');
+const { print } = require('graphql');
+const {
+  addMockFunctionsToSchema,
+  introspectSchema,
+  makeExecutableSchema,
+  makeRemoteExecutableSchema,
+  mergeSchemas,
+} = require('graphql-tools');
+const fetch = require('isomorphic-fetch');
+
+/* eslint-disable node/no-missing-require */
+const {
+  schemas = [],
+  resolvers = [],
+  mocks = {},
+} = require('hops-graphql-mocks');
```

Comment:
Add a comment to describe that this is an alias to the `graphqlMocks` preset config option.